Django-CoWork-Connect/coapp/admin.py

```diff
@@ -28,3 +28,7 @@ def cancel_reservations(self, request, queryset):
 @admin.register(OurOffer)
 class OurOfferAdmin(admin.ModelAdmin):
```

Comment:
te klasy sa niepotrzebne jezeli sa puste po prostu
```suggestion
@admin.register(OurOffer)
```